ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,225 @@
+# Alzadaal's Legacy
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# SILKIE #
+#~~~~~~~~#
+
+# -ii 77ED 7760 7761 7762 7768 7769 776A
+
+0.0 "--sync--" sync / 00:0839::The Trial of Knowledge is sealed off/ window 0,1
+12.2 "--sync--" sync / 14:[^:]*:Silkie:7759:/ window 15,15
+14.9 "Fizzling Suds" sync / 1[56]:[^:]*:Silkie:7759:/
+21.0 "Soap's Up" sync / 1[56]:[^:]*:Silkie:775A:/
+22.0 "Fizzling Duster" sync / 1[56]:[^:]*:Silkie:775D:/
+32.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+38.4 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+43.5 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+50.1 "--sync--" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+51.6 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+53.3 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+54.8 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+61.9 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+64.0 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+69.0 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+73.0 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+80.6 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+89.7 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+89.7 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+89.7 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+89.7 "Chilling Duster?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
```

Comment:
```suggestion
89.7 "Bracing Duster / Chilling Duster / Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:(7769|7768|776A):/
```

Reply:
It is possible to write in some branching for Silkie since we can branch from the Suds cast then rejoin the main timeline.

Reply:
I think branching would be the ideal goal, but I didn't code it in for simplicity.

Reply:
I'm not sure this is complicated enough to merit the branching, so keeping it simple for now seems good.

Reply:
Ok, I thought this one out some more.
There will always be a Bracing Duster and a Chilling Duster when resolved correctly. The last one has to be Chilling or Fizzling as two Bracing Dusters results in a guaranteed damage down. I have fixed it to show the "correct" three lines. There will always be two Bracing Dusters prior to making a Chilling Duster with Slippery Soap given the boss always has Bracing Suds here and converts the north duster + east or west.